fipy/meshes/abstractMesh.py

```diff
@@ -692,6 +692,80 @@ def _localOverlappingFaceIDs(self):
         """
         return self.topology._localOverlappingFaceIDs
 
+    @property
+    def _vertexCellIDs(self):
+        """Return cell IDs bounded by each vertex
+
+        Returns
+        -------
+        masked_array
+
+        Examples
+        --------
+
+        ```
+        6-------7-------8
+        |       |       |
+        |   2   |   3   |
+        |       |       |
+        3-------4-------5
+        |       |       |
+        |   0   |   1   |
+        |       |       |
+        0-------1-------2
+        ```
+
+        >>> from fipy.meshes import Grid2D
+        >>> m = Grid2D(nx=2, ny=2)
+        >>> print(m._cellVertexIDs) # doctest: +SERIAL
+        [[1 2 4 5]
+         [4 5 7 8]
+         [3 4 6 7]
+         [0 1 3 4]]
+        >>> print(m._vertexCellIDs) # doctest: +SERIAL
+        [[0 0 1 0 0 1 2 2 3]
+         [-- 1 -- 2 1 3 -- 3 --]
+         [-- -- -- -- 2 -- -- -- --]
+         [-- -- -- -- 3 -- -- -- --]]
+        """
```

Comment:
Sorry, I do not understand how to get from the mesh to the array here; perhaps due to lack of familiarity with masked arrays?
The mapping of the vertices bounding each cell makes sense, though the order seems strange (vertices surrounding cell 1, 3, 2, 0, in counter-clockwise order).
@guyer, can you explain what the `_vertexCellIDs` means to me? Again, sorry... this looks like a simple accessor function, so I understand if the question is out of scope for this PR.

Reply:
The [SO thread](https://stackoverflow.com/questions/59669831/method-for-cells-adjacent-connected-to-vertex-in-fipy/59686318#59686318) has `_vertexToCellIDs` as an `nCell`-length of length-4 lists, each containing the IDs cells with which the vertex is associated, up to the possible 4, padded by `-1`. This seems to make sense.
I'm unclear how that representation maps to this one.

Reply:
@guyer/@wd15, nevermind -- I've got it.